src/os/ui/ogc/ogcserver.js

```diff
@@ -983,6 +1061,11 @@ os.ui.ogc.OGCServer.prototype.parseWfsCapabilities = function(response, uri) {
         descriptor.setKeywords(tags);
       }
 
+      if (descriptor instanceof os.data.BaseDescriptor) {
```

Comment:
This feels like it's working around API design a bit. Should the new functions be added to an interface instead of using `instanceof`? I'm not sure if they're intended to be specific to OGC, making `IOGCDescriptor` an appropriate place for them, or if other descriptors may also benefit which would make `IDataDescriptor` a better choice.